### DISCUSSION TITLE
Table of Contents in Pdf file created from convert

### DISCUSSION THREAD

**Discussion by xs2ahmed** [ImageMagick/ImageMagick#2359] [Help]
Hi, 
I am using convert command to convert directory of images to pdf file.
convert chapter1/ *.jpg chapter2/ *.jpg book.pdf
It creates a pdf fine, 
How can I create table of contents? for easier access to chapters while viewing the pdf ?? Please help this is urgent. Thanks alot guys

**Comment by fmw42** [comment]
ImageMagick convert will not create a table of contents. You will  have to make that file separately and then put that file in your command before the chapters. You can create a table of contents file using one of the text to image commands. See https://imagemagick.org/Usage/text/

**Reply by xs2ahmed** [reply]
I think I could not convey properly what I wanted to do, I will give it another try. I have attached a screen shot below, When I open this book in document viewer or adbode reader I get that outline and table of contents on the left pane. So when I click on each item in the left pane it takes me to a certain page in the pdf file. That is what I want to achieve, the purpose is not to just create a page before chapters but easy to access menu on the left as you see in the screen shot. 
![Screenshot from 2020-08-03 22-53-39](https://user-images.githubusercontent.com/7674266/89212044-39f37780-d5dc-11ea-98c5-d23f1b2b34eb.png)

**Comment by fmw42** [comment]
ImageMagick cannot create such a PDF. It can only make the pages. It has no ability to link pages to a table of contents.
